src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -170,6 +173,46 @@ object VEP {
       a(4).split(","))
   }
 
+  def getCSQHeaderDefinition(cmd: Array[String], perl5lib: String, path: String): String = {
+    val csq_header_regex = "ID=CSQ[^>]+Description=\"([^\"]+)".r
+    val pb = new ProcessBuilder(cmd.toList.asJava)
+    val env = pb.environment()
+    if (perl5lib != null)
+      env.put("PERL5LIB", perl5lib)
+    if (path != null)
+      env.put("PATH", path)
+    val (jt, proc) = List(Variant("1", 13372, "G", "C")).iterator.pipe(pb,
+      printContext,
+      printElement,
+      _ => ())
+
+    val csq_header = jt.flatMap(s => csq_header_regex.findFirstMatchIn(s).map(m => m.group(1)))
+    val rc = proc.waitFor()
+    if (rc != 0)
+      fatal(s"vep command failed with non-zero exit status $rc")
+
+    if (csq_header.hasNext)
+      csq_header.next()
+    else {
+      warn("Could not get VEP CSQ header")
+      ""
+    }
+  }
+
+  def getNonStarAlleleMap(v: Variant): mutable.Map[Int, Int] = {
```

Comment:
I'd change the name to indicate what map goes to/from.  `getNonStarToOriginalAlleleIdxMap`.